### PR TITLE
Include file option description in token, session-id parameters

### PR DESCRIPTION
**Pull request by stevenGravy**
Provides description that a file path is also a option for:
- token
- session id

Will update docs if this is approved.

### CODE REVIEW

File: tool/tbot/main.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func Run(args []string, stdout io.Writer) error {
 
 	startCmd := app.Command("start", "Starts the renewal bot, writing certificates to the data dir at a set interval.")
 	startCmd.Flag("auth-server", "Address of the Teleport Auth Server or Proxy Server.").Short('a').Envar(authServerEnvVar).StringVar(&cf.AuthServer)
-	startCmd.Flag("token", "A bot join token, if attempting to onboard a new bot; used on first connect.").Envar(tokenEnvVar).StringVar(&cf.Token)
+	startCmd.Flag("token", "A bot join token or full path to file with token value, if attempting to onboard a new bot; used on first connect.").Envar(tokenEnvVar).StringVar(&cf.Token)
```
Comment (zmb3) [inline tool/tbot/main.go:72]: "Full path" is not as clear as _absolute path_.

But I wonder, does it actually have to be an absolute path?

Comment (stevenGravy) [reply tool/tbot/main.go:72]: We tend to use "Full Path" or just path.  You can do relative paths AFAIK, just that if you're running in a service you better run it in the right place so full path recommended I'd say.  Only error messages on the term absolute I could fine.

`--pid-file            Full path to the PID file. By default no PID file will be created`
or just path
`  -c, --config              Path to a configuration file [/etc/teleport.yaml]`

Comment (stevenGravy) [reply tool/tbot/main.go:72]: Please see comments @zmb3 when you get a chance

File: tool/teleport/common/teleport.go [hunk 2]
```diff
@@ -132,7 +132,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 		fmt.Sprintf("Address of the auth server [%s]", defaults.AuthConnectAddr().Addr)).
 		StringsVar(&ccf.AuthServerAddr)
 	start.Flag("token",
-		"Invitation token to register with an auth server [none]").
+		"Invitation token or full path to file with token value. Used to register with an auth server [none]").
```
Comment (zmb3) [inline tool/teleport/common/teleport.go:135]:
```suggestion
		"Invitation token or path to file with token value. Used to register with an auth server [none]").
```

**Review by zmb3 — APPROVED** [review]

**Review by AntonAM — APPROVED** [review]

**Pull request merged**